src/System.Net.Mime/tests/Unit/Base64EncodingTest.cs

```diff
@@ -0,0 +1,179 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+using System.Text;
+using Xunit;
+
+namespace System.Net.Mime.Tests
+{
+    public class Base64EncodingTest
+    {
+        [Theory]
+        [InlineData("some test header to base64 encode")]
+        [InlineData("some test header to base64 � �� asdf�� encode")]
```

Comment:
nit: Our current best-practice is to use \uXXXX for unicode characters instead of using the actual characters.  I know this is a port of existing code. But something to change in the future.